content/news/2023-11-17-bwhpc-symposium/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: bwHPC Symposium
+date: '2023-11-17'
```

Comment:
Date back a week after the event maybe?

Reply:
Done.

Reply:
Wait, does that also mean to predate the directory name? If so I will have to update the directory name,

Reply:
I think so, not really sure how the hub is handling the file name vs. the annotation. Something to try out locally.